bin/functional-tests/test_config.py

```diff
@@ -33,7 +33,10 @@ def test_houston_config(houston_api):
             f"Expected not to find 'localhost' in the 'servers' configuration. Found:\n\n{houston_config['nats']}"
 
 def test_houston_can_reach_prometheus(houston_api):
-    houston_api.check_output("wget --timeout=1 http://astronomer-prometheus.astronomer.svc.cluster.local:9090/targets")
+    houston_api.check_output("wget -qO- --timeout=1 http://astronomer-prometheus.astronomer.svc.cluster.local:9090/targets")
+
+def test_nginx_can_reach_default_backend(nginx):
+    nginx.check_output("curl --max-time 1 http://astronomer-nginx-default-backend:8080")
```

Comment:
super tiny nit. Do you want to add `-s` here for silent?

Reply:
Also, why wget in one instance and wget in another? It seems like we should choose one method for doing these checks and stick with it.

Reply:
because of what is available in the container